content/releases/3-0-0rc08.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 3.0.0rc08
+date: '2023-12-19T13:01:02-05:00'
+description: Release notes for v3.0.0rc08 of the Rhize application
+categories: ["releases"]
+weight: 1 ##one if latest
+menu:
+  main:
+    parent: releases
+---
+
+Release notes for version 3.0.0rc08 of the Rhize application.
+
+_Release date:_ 19th December, 2023
+
+## Breaking changes
+
+  - [ADMINUI] Change UI navigation bar design
+  - [SCHEMA] Change OperationsEventClass.IncludedIn to reference an OperationsEventClassVersion instead of OperationsEventClass
+  - [SCHEMA] Change OperationsEventClass.IncludesPropertiesOf to move to the version instance, OperationsEventClassVersion.IncludesPropertiesOf, instead of header 
+  - [SCHEMA] Change OperationsEventDefinition.IncludedIn to reference an OperationsEventDefinitionVersion instead of OperationsEventDefinition
+  - [SCHEMA] Change OperationsEventDefinition.IncludesPropertiesOf to move to the version instance, OperationsEventDefinitionVersion.IncludesPropertiesOf, instead of header
+  - [SCHEMA] Change FromResourceReference and ToResourceReference to combine them into a single resourceReference

```

Comment:
```suggestion
  - [SCHEMA] Change `OperationsEventClass.IncludedIn` to reference an `OperationsEventClassVersion` instead of `OperationsEventClass`
  - [SCHEMA] Change `OperationsEventClass.IncludesPropertiesOf` to move to the version instance, `OperationsEventClassVersion.IncludesPropertiesOf`, instead of header 
  - [SCHEMA] Change `OperationsEventDefinition.IncludedIn` to reference an `OperationsEventDefinitionVersion` instead of `OperationsEventDefinition`
  - [SCHEMA] Change OperationsEventDefinition.IncludesPropertiesOf to move to the version instance, OperationsEventDefinitionVersion.IncludesPropertiesOf, instead of header
  - [SCHEMA] Change FromResourceReference and ToResourceReference to combine them into a single resourceReference

```